app/views/courses/confirmation.html.erb

```diff
@@ -153,12 +175,34 @@
               </dd>
             </div>
 
+            <% if !@provider.accredited_body? %>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">
+                  Accredited body
+                </dt>
+                <dd class="govuk-summary-list__value" data-qa="course__accredited_body">
+                  <%= course.accrediting_provider&.provider_name %>
```

Comment:
If it's not an accredited body it should have an accrediting provider right? So why not 
```suggestion
                  <%= course.accrediting_provider.provider_name %>
```